Code/GraphMol/MolInterchange/Parser.cpp

```diff
@@ -157,11 +164,14 @@ std::string getStringDefaultValue(const char *key, const rj::Value &from,
 }
 
 void readAtom(RWMol *mol, const rj::Value &atomVal,
-              const DefaultValueCache &atomDefaults) {
+              const DefaultValueCache &atomDefaults,
+              const JSONParseParameters &params) {
   PRECONDITION(mol, "no mol");
   Atom *at = new Atom(getIntDefaultValue("z", atomVal, atomDefaults));
```

Comment:
do you mean the JSON field names like `z`?
That seems like adding overhead for constants which are only used twice (once in the reader and once in the writer) and which won't be used elsewhere

Reply:
formatVersion is used at least three times.  I’m not sure if the compiler is smart enough to have these in the text section only once, but this has just been a general optimization I’ve learned the hard way, imho it’s always better to make things compile time errors over run time

Reply:
> imho it’s always better to make things compile time errors over run time
Sure, but that's not what this is. The compiler isn't going to help me if I put the wrong constant in my code (e.g. `CONST_Y` instead of `CONST_Z`) any more than it'll help if I put `"y"` instead of `"z"`.  That's what the writer/parser tests are for.

I think it's overhead, and it's a pattern which occurs everywhere in the code.